docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+# How To Upgrade Your Subnet-EVM
+
+## Upgrading a Subnet
+
+You can parametrize Subnet-EVM based Subnets after deployment by enabling and disabling precompiles.
+To do this, create a `upgrade.json` file and place in the appropriate directory. 
+
+This page is a tutorial on how to perform such network upgrades. It's specific for Subnet-EVM upgrades.
+
+
+[Upgrade a Subnet](./subnet-upgrade) 
+describes all the background information required regarding Subnet upgrades.
+
+:::warning
+
+It's very important that you read and understood the previously linked document. 
+Failing to do so can potentially grind your network to a halt.
+
+
+:::
+
+This tutorial assumes that you already 
+[installed](./install-avalanche-cli) Avalanche-CLI.
+It also uses `testSubnet` as a hypothetical Subnet name throughout.
+
+
+## Generate the Upgrade File
+
+The 
+[Precompiles](./customize-a-subnet#network-upgrades-enabledisable-precompiles) 
+documentation describes what files the network upgrade requires, and where to place them.
+
+To generate a valid `upgrade.json` file, run:
+
+```shell
+avalanche subnet upgrade generate testSubnet
+```
+
+If you didn't create `testSubnet` yet, you would see this result:
+
+```shell
+avalanche subnet upgrade generate testSubnet
+The provided subnet name "testSubnet" does not exist
+```
+
+Again, it makes no sense to try the upgrade command if the Subnet doesn't exist. 
+If that's the case, please go ahead and 
+[create](./build-first-subnet) the Subnet first.
+
+If the Subnet definition exists, the tool launches a wizard.
+It may feel a bit redundant, but you first see some warnings, to draw focus to the dangers involved:
+
+```shell
+avalanche subnet upgrade generate testSubnet       
+Performing a network upgrade requires coordinating the upgrade network-wide.
+A network upgrade changes the rule set used to process and verify blocks, such that any node that 
+upgrades incorrectly or fails to upgrade by the time that upgrade goes into effect may become 
+out of sync with the rest of the network.
+
+Any mistakes in configuring network upgrades or coordinating them on validators may cause the 
+network to halt and recovering may be difficult.
+Please consult 
+https://docs.avax.network/subnets/customize-a-subnet#network-upgrades-enabledisable-precompiles 
+for more information
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Press [Enter] to continue, or abort by choosing 'no': 
+  ▸ Yes
+    No
+```
+
+Go ahead and select `Yes` if you understood everything and you agree.
+
+You see a last note, before the actual configuration wizard starts:
```

Comment:
```suggestion
The follow messages will be shown before the actual configuration wizard starts:
```

Reply:
This will result in the "passive voice" warning...I already did a pass yesterday eliminating all passive voice parts in the PR